lib/measured/units/dimension.rb

```diff
@@ -0,0 +1,26 @@
+class Measured::Dimension < Measured::Measurable
+
+  conversion.set_base :m,
+    aliases: [:meter, :metre, :meters, :metres]
+
+  conversion.add :cm,
+    aliases: [:centimeter, :centimetre, :centimeters, :centimetres],
+    value: "0.01 m"
+
+  conversion.add :mm,
+    aliases: [:millimeter, :millimetre, :millimeters, :millimetres],
+    value: "0.001 m"
+
+  conversion.add :in,
+    aliases: [:inch, :inches],
+    value: "0.0254 m"
+
+  conversion.add :ft,
+    aliases: [:foot, :feet],
+    value: "0.3048 m"
+
+  conversion.add :yd,
+    aliases: [:yard, :yards],
+    value: "0.9144 m"
+
+end
```

Comment:
totally doesn't matter for the purposes of this project, but someone's going to want to extend this into kilometers and miles in the future